fastlane/assets/render_plugin.md.erb

```diff
@@ -2,7 +2,7 @@
 
 <div class="quality-index" style="color: #<%= @plugin.color_to_use %>"><%= @plugin.data[:overall_score] %></div>
 
-<h3><a href="<%= @plugin.homepage %>" target="_blank"><code><%= @plugin.name.gsub("fastlane-plugin-", "") %></code></a></h3>
+<h3 id="<%= @plugin.name.gsub("fastlane-plugin-", "") %>"><a href="<%= @plugin.homepage %>" target="_blank"><code><%= @plugin.name.gsub("fastlane-plugin-", "") %></code></a></h3>
```

Comment:
Is it preferred to make this a `name` attribute on the anchor? This way it will update the navigation bar to have the named anchor in the url when clicked? 🤔

```suggestion
<h3><a  name="<%= @plugin.name.gsub("fastlane-plugin-", "") %>" href="<%= @plugin.homepage %>" target="_blank"><code><%= @plugin.name.gsub("fastlane-plugin-", "") %></code></a></h3>
```

Reply:
Does it do that? I think this only happens if it is an empty anchor with `href="#"` or similar, not for an actual link that open a site.

Reply:
Ahhhhhhh, good point! Your implementation is 👍

Reply:
Hmm, for normal headlines we get his "header anchor" that appears left to the actual headline. This is the code for it:
![image](https://user-images.githubusercontent.com/183673/48900574-0518ae80-ee53-11e8-8a6a-712ab5b919b3.png)
But I couldn't find out where this comes from really - but it would be perfect to be able to hotlink to the individual plugins as well!

Reply:
Oh, it's an extension to mkdocs:
https://github.com/fastlane/docs/search?q=headeranchor&unscoped_q=headeranchor

So why isn't this applied to the `h3` headline of the plugins?